pkg/gds/store/trtl/iterator_test.go

```diff
@@ -345,3 +348,57 @@ func (s *trtlStoreTestSuite) TestStreamingIterator() {
 	require.Nil(iter.Error())
 	iter.Release()
 }
+
+func (s *trtlStoreTestSuite) TestStreamingIteratorError() {
+	require := s.Require()
+	iter := trtl.NewTrtlStreamingIterator(&trtlErrorClient{}, "")
+
+	// Call interactive methods on iter should do nothing but not return an error.
```

Comment:
```suggestion
	// Calling interactive methods on iter should do nothing but not return an error.
```

Reply:
Ah shoot, I went too fast and I didn't manage to merge the suggestion; will update in the next PR - sorry.